aten/src/ATen/native/RNN.cpp

```diff
@@ -138,6 +166,52 @@ struct QuantizedCellParams {
   }
 };
 
+struct QuantizedCellParamsDynamic {
```

Comment:
Is the purpose of this struct that we do not require explicit scale and zero point parameters? You should add a comment here explaining why we have both `QuantizedCellParams` and `QuantizedCellParamsDynamic`

Reply:
That is right. Basically the Tensor in the arguments encapsulates the scale and the zero point in the struct in https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/quantized/cpu/fbgemm_utils.h#L18-L23. I will add a comment here.